pkg/scheduler/reconciliation/filter.go

```diff
@@ -8,12 +8,106 @@ import (
 	"github.com/kyma-incubator/reconciler/pkg/model"
 )
 
+type FilterMixer struct {
```

Comment:
Would be good to have a unit-test for it.

You can get a generated DB string from the query object (`q.String()`)

Reply:
Good idea